src/main/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/query/impl/expression/MarkerExpression.java

```diff
@@ -0,0 +1,18 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.impl.expression;
+
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.CypherExpression;
+
+public class MarkerExpression extends AliasedExpression {
```

Comment:
Can you please add a JavaDoc comment that explains the purpose of this type of expression.